### ISSUE TITLE
Restart vasp calculation

### ISSUE THREAD

**Issue by nkeilbart** [prisms-center/CASMcode#187]
I tried to find an email list to send this question and found an old email list so I will post my question here. 

I have recently worked my way through the example problem and started working on my project of interest. I was able to generate all the files and submit the files for calculation. One of the configurations failed to converge due to needing a lower threshold. I am currently attempting to restart this calculation using the CASM interface. I am aware I could manually change the files to the correct format after manually restarting the job but I imagine there is a better way. 

If there is a better place for this please let me know and I will post it there. I have really enjoyed the software and appreciate the effort. Thank you.

**Comment by bpuchala** [comment]
The `casm format --vasp` help output shows where you can include another "settings" directory that will get checked to override settings on a configuration-by-configuration basis:

```
DESCRIPTION:
CASM comes with wrappers for using VASP to calculate the properties 
of configurations, but is designed so that any type of calculation  
software or method could be used if an appropriate set of wrapper   
scripts are available. By convention, input settings for software   
used to calculate the properties of a particular configuration      
should be checked for in the following directories:                 
  1) $ROOT/training_data/$SCELNAME/$CONFIGID/settings/$CURR_CALCTYPE
  2) $ROOT/training_data/$SCELNAME/settings/$CURR_CALCTYPE          
  3) $ROOT/training_data/settings/$CURR_CALCTYPE
```

You can use this to customize the input or a particular configuration, up the run limit in relax.json, etc. When you re-submit this calculation through the `casm calc --submit` interface it will use any input files (including relax.json) placed in `$ROOT/training_data/$SCELNAME/$CONFIGID/settings/$CURR_CALCTYPE` before it uses the ones placed in `$ROOT/training_data/settings/$CURR_CALCTYPE`.

I got the notification of your subscription to the email list. Either way to post questions is OK.

**Comment by nkeilbart** [comment]
Thanks for getting back to me. I will try this out but otherwise you can consider this issue closed.

**Comment by nkeilbart** [comment]
Actually, I was attempting this and get the following status when attempting to submit.

Construct a casm.vaspwrapper.Relax instance:
  Input directory: /p/lustre2/keilbart/CASM_Test/training_data/SCEL1_1_1_1_0_0_0/4
  Configuration: SCEL1_1_1_1_0_0_0/4
  Reading CASM settings
  Calculations directory: /p/lustre2/keilbart/CASM_Test/training_data/SCEL1_1_1_1_0_0_0/4/calctype.default
  Reading relax.json settings file
  Read settings from: /p/lustre2/keilbart/CASM_Test/training_data/SCEL1_1_1_1_0_0_0/4/settings/calctype.default/relax.json
  DONE

Submitting...
Configuration: SCEL1_1_1_1_0_0_0/4
Calculation directory: /p/lustre2/keilbart/CASM_Test/training_data/SCEL1_1_1_1_0_0_0/4/calctype.default
JobID: ['5584890']
JobID: 5584890   Jobstatus: ?   Not submitting.

So it finds the updated settings that I have within the individual configuration folder. I use casm-calc --setup which puts the files in the right folder and then I run the casm-calc --submit which shows the above output. It then seems to find the previous job ID within the database file created by CASM and then does not resubmit the job. Am I supposed to somehow purge this somewhere or is there an additional command to use?

**Comment by nkeilbart** [comment]
As an update, I found the jobs.db file in the hidden .prisms_jobs folder. I managed to open up the database, delete the entry for the job, also delete the files for the configuration in the training_data folder, ran casm-calc --setup and then casm-calc --submit which is then able to resubmit the job. 

So I've found a work around but I ask once again, is there a way that CASM can do this already. I've dug around in the code and didn't notice anything right away but also have not dedicated too much time to that. I appreciate the help.
